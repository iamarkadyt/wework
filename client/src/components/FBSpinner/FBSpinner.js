import React from 'react'
import './FBSpinner.scss'

const fBSpinner = () => (
    <div className="lds-facebook">
        <div>
        </div>
        <div>
        </div>
        <div>
        </div>
    </div>
)

export default fBSpinner
