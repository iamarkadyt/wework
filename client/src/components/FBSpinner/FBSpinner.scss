.lds-facebook {
    display: inline-block;
    position: relative;
    width: 3.2rem;
    height: 3.2rem;
    transform: scale(.7);
    margin: 1.5rem 0;
}

.lds-facebook div {
    display: inline-block;
    position: absolute;
    left: .3rem;
    width: .65rem;
    background: rgba($color: #ffffff, $alpha: 0.15);
    animation: lds-facebook 1.2s cubic-bezier(0, 0.5, 0.5, 1) infinite;
}

.lds-facebook div:nth-child(1) {
    left: .3rem;
    animation-delay: -0.24s;
}

.lds-facebook div:nth-child(2) {
    left: 1.3rem;
    animation-delay: -0.12s;
}

.lds-facebook div:nth-child(3) {
    left: 2.25rem;
    animation-delay: 0;
}

@keyframes lds-facebook {
    0% {
        top: .3rem;
        height: 2.55rem;
    }
    50%,
    100% {
        top: .95rem;
        height: 1.3rem;
    }
}