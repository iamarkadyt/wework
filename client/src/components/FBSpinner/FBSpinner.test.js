import FBSpinner from './FBSpinner'
import { shallow } from 'enzyme'
import React from 'react'

const comp = () => {
  return shallow(<FBSpinner />)
}

it('matches snapshot', () => {
  expect(comp()).toMatchSnapshot()
})
