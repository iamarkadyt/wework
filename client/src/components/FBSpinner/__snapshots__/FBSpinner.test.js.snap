// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div
  className="lds-facebook"
>
  <div />
  <div />
  <div />
</div>
`;
